spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index Page', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Doraemon & Nobita', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Best friends', posts_counter: 12)
+    @post = Post.create(title: 'Physics', text: 'This is not my first post', comments_counter: 10, likes_counter: 10,
+                        author: @user)
+  end
+
+  describe 'User show page' do
+    before(:each) { visit user_path(id: @user.id) }
+
+    # it 'displays a container for the users' do
+    #   expect(page).to have_css('section.single_users_container')
+    # end
```

Comment:
- [ ] it would be better to remove the uncommented code in order to keep the file clean and pro